x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/resource_tab.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiAccordion,
+  EuiCode,
+  EuiCodeBlock,
+  EuiDescriptionList,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { getFlattenedObject } from '@kbn/std';
+import { CspFinding } from '../types';
+import * as TEXT from '../translations';
+
+const getDescriptionDisplay = (value: any) => {
+  if (typeof value === 'boolean' || value === undefined || value === null) {
+    return <EuiCode>{JSON.stringify(value)}</EuiCode>;
+  }
+
+  if (typeof value === 'object') {
+    return (
+      <EuiCodeBlock isCopyable={true} overflowHeight={300}>
+        {JSON.stringify(value, null, 2)}
+      </EuiCodeBlock>
+    );
+  }
+
+  return <EuiText size="s">{value}</EuiText>;
+};
+
+const prepareDescriptionList = (data: Record<string, any>) =>
+  Object.entries(getFlattenedObject(data))
+    .sort((a, b) => a[0].localeCompare(b[0]))
+    .map(([key, value]) => ({
+      title: (
+        <EuiText size="s">
+          <strong>{key}</strong>
+        </EuiText>
+      ),
+      description: getDescriptionDisplay(value),
+    }));
+
+export const ResourceTab = ({ data }: { data: CspFinding }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const accordions = useMemo(
+    () => [
+      {
+        title: TEXT.RESOURCE,
+        id: 'resourceAccordion',
+        listItems: prepareDescriptionList(data.resource),
+      },
+      {
+        title: TEXT.HOST,
+        id: 'hostAccordion',
+        listItems: prepareDescriptionList(data.host),
+      },
+    ],
+    [data.host, data.resource]
+  );
+
+  return (
+    <>
+      {accordions.map((accordion) => (
+        <>
```

Comment:
No need for double fragment, you can simply do:

```
return accordions.map((accordion) => (
    <>
      <EuiPanel hasShadow={false} hasBorder>
```

Reply:
not sure this is correct, this change makes this function to just return an array which then yells:
```
TS2786: 'ResourceTab' cannot be used as a JSX component. 
  Its return type 'Element[]' is not a valid JSX element.    
    Type 'Element[]' is missing the following properties from type 'ReactElement<any, any>': type, props, key
```